File: atomicpi/samples/gpio-test.js
var atomicpi = require("atomicpi");
var GPIO = require("sysfs-gpio");
// Control by signal ID
GPIO.export(atomicpi.signals.ISH_GPIO_0.global_idx, (pin) => {
   pin.out();
   pin.high();
});
// Control with signal ID lookup on Enchilada connector first
GPIO.export(atomicpi.signals[atomicpi.connectors.enchilada.leds.green].global_idx, (pin) => {
   pin.out();
   pin.low();
});
process.stdin.resume();

File: atomicpi/samples/gpio-test.py
import atomicpi
import gpio as GPIO
# Control by signal ID
GPIO_0=atomicpi.signals.ISH_GPIO_0.global_idx
GPIO.setup(GPIO_0, GPIO.OUT)
GPIO.output(GPIO_0, True)
# Control with signal ID lookup on Enchilada connector first
GREEN_LED=atomicpi.signals[atomicpi.connectors.enchilada.leds.green].global_idx
GPIO.setup(GREEN_LED, GPIO.OUT)
GPIO.output(GREEN_LED, False)

GPIO.cleanup(GPIO_0)
GPIO.cleanup(GREEN_LED)
